blockchain/chainio.go

```diff
@@ -479,8 +482,13 @@ func serializeSpendJournalEntry(stxos []spentTxOut) ([]byte, error) {
 	var size int
 	var sizes []int
 	for i := range stxos {
-		// size += spentTxOutSerializeSize(&stxos[i])
 		sz := spentTxOutSerializeSize(&stxos[i])
+		if sz < minimumStxoSize {
```

Comment:
It doesn't look like you can ever trigger this error, so I ended up just removing it.